### PR TITLE
Adopt to merge of the servers

### PR DESCRIPTION
**Pull request by dacrome**
Also cleanup all changed files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,39 +11,39 @@ create a new file named "addon-administration.properties" in that directory
...
-# displayed instead of the real extension field name. Each item should follow
-# this pattern: <extension-urn>::<extension field name>=<Display value>
-org.osiam.administration.extensions=\
-urn:org.osiam:scim:extensions:tests::age=Your age
-```
+## Configuration
+
+### OSIAM Endpoints
+    org.osiam.endpoint=http://<osiam-tomcat>/osiam
+    org.osiam.redirectUri=http://<admin-tomcat>/addon-administration/
```
Comment (tkrille) [inline README.md]: > admin-tomcat
???

Comment (tkrille) [inline README.md]: Oh, I see. Why tomcat? "osiam address", "addon-self-administration address"?

File: src/main/sql/admin_group.sql [hunk 2]
```diff
@@ -1,6 +1,6 @@
 --
 -- Group 'admin' for the addon-administration. Has to be imported in the
--- database of the resource-server.
+-- database of osiam.
```
Comment (tkrille) [inline src/main/sql/admin_group.sql]: "OSIAM", please.

File: README.md [hunk 3]
```diff
@@ -80,8 +80,8 @@ the above mentioned addon-administration.properties!
 
 ### Admin group
 
-You need to add a specific group for administration in the resource-server
-database (admin_group.sql).
+You need to add a specific group for administration in the osiam database
```
Comment (tkrille) [inline README.md]: "OSIAM", please.

File: README.md (hunk 1 above)
Comment (dacrome) [inline README.md]: Done

File: src/main/sql/admin_group.sql (hunk 2 above)
Comment (dacrome) [inline src/main/sql/admin_group.sql]: Done

File: README.md (hunk 3 above)
Comment (dacrome) [inline README.md]: Done

**Comment by tkrille** [comment]
LGTM, waiting for https://github.com/osiam/osiam/pull/44 to merged.

**Pull request merged**